From 3798babc04e344cde25068aad8e6b071c0fe20b1 Mon Sep 17 00:00:00 2001
From: blunden <blunden2@gmail.com>
Date: Fri, 14 Dec 2012 03:59:13 +0100
Subject: [PATCH 11/28] LatinIME: Enable spellchecker for additional languages

We include more dictionaries than stock builds so let's
also enable spell checking for languages that use only
the supported unicode characters.

Change-Id: Iff9aea56edde10e47a754df39682f4f72c269756
---
 java/res/xml/spellchecker.xml | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/java/res/xml/spellchecker.xml b/java/res/xml/spellchecker.xml
index d2da17f9d..119e69947 100644
--- a/java/res/xml/spellchecker.xml
+++ b/java/res/xml/spellchecker.xml
@@ -63,4 +63,20 @@
             android:label="@string/subtype_generic"
             android:subtypeLocale="hu"
     />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="da"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="fi"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="nb"
+    />
+    <subtype
+            android:label="@string/subtype_generic"
+            android:subtypeLocale="sv"
+    />
 </spell-checker>
-- 
2.17.1

